src/xenia/gpu/gpu_flags.cc

```diff
@@ -20,6 +20,8 @@ DEFINE_path(
 
 DEFINE_bool(vsync, true, "Enable VSYNC.", "GPU");
 
+DEFINE_uint64(vsync_interval, 16, "VSYNC interval.", "GPU");
```

Comment:
I would add another note about the used unit. Like "[...] in milliseconds."
Otherwise people would assume its fps and complain why its broken...